Dockerfile

```diff
@@ -19,11 +21,10 @@ ADD rel/ rel/
 RUN mix distillery.release --verbose
 
 # The one the elixir image was built with
-FROM debian:stretch
+FROM alpine:3.11
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-		libssl1.1 libsctp1 curl dumb-init \
-	&& rm -rf /var/lib/apt/lists/*
+RUN apk add --update libssl1.1 curl bash dumb-init \
```

Comment:
No more need for `libsctp1`? How did you determine that? @ianwestcott and I had a discussion once about these, and best we could come up with was just some trial and error with different combinations of them to see if the app barfs.

Reply:
Pretty much. I started with only libssl1.1 and dumb-init since I know those are required, and added curl and bash back because the app complained.

Reply:
This question took me down a fun little rabbit hole which led here: https://bugs.erlang.org/browse/ERL-262?focusedCommentId=11711&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel

TL;DR: The dependency on `libsctp1` was bogus and was caused by a bug in OTP 19.1 that was later fixed. It's possible that we (or even the Debian packages?) still assume it's required, but Alpine doesn't.